docs/fleet/provision.md

```diff
@@ -1,278 +1,245 @@
-   ```
-
-{{% /tab %}}
-{{% tab name="Install using a configuration file" %}}
-
-If you want to install the Viam Agent on a machine that you have not yet created in the Viam app, follow these steps:
-
-1. Create a configuration file with the desired configuration for your machine. You can:
-
-   - You can [create a new machine in the Viam app](/cloud/machines/#add-a-new-machine) and configure it as desired, then switch to **Raw JSON** mode and copy the configuration shown into a new file on your machine.
-   - You can base your configuration on the [example configuration file](/internals/local-configuration-file/#example-json-configuration-file), and adjust as needed.
-   - You can use an existing configuration file from a deployed machine, or adapt such a file as needed to fit the specifications of your new machine.
-
-1. Place this file in the following location on the machine you wish to install the Viam Agent to: <file>/etc/viam.json</file>
-
-1. Run the following command to install the Viam Agent on your machine:
+## End user experience
 
-   ```sh {class="command-line" data-prompt="$"}
-   sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/install.sh)"
-   ```
+End users receive a machine, and use either a captive web portal or mobile app to complete the machine setup.
 
-{{% /tab %}}
-{{< /tabs >}}
+One option is to use the [Viam mobile app](/fleet/control/#control-interface-in-the-viam-mobile-app).
+The Viam mobile app allows end users to create a new machine in the app, and `agent-provisioning` will then install `viam-server` and run it with a provided configuration.
 
-### Update the Viam Agent and `viam-server`
+To add your branding, you can build your own mobile app and use the [Flutter SDK](https://flutter.viam.dev/viam_protos.provisioning.provisioning/ProvisioningServiceClient-class.html) or the [TypeScript SDK](https://github.com/viamrobotics/viam-typescript-sdk/blob/main/src/app/provisioning-client.ts) to connect to `viam-agent` and provision your machines.
 
-The Viam Agent automatically updates both itself and `viam-server` as new updates are released.
-You can also configure update behavior for the Agent and `viam-server` using the [Viam app](https://app.viam.com/).
+If you are not using Flutter or TypeScript and would like to use provisioning, please [contact us](mailto:support@viam.com).
 
-{{< alert title="Important" color="note" >}}
-When the Viam Agent updates either itself or `viam-server`, you must restart these services in order to use the new version.
-When you stop or restart the Viam Agent, the agent will stop or restart `viam-server` as well.
-{{< /alert >}}
+This is the general process for provisioning depending on whether you are using a captive web portal or a mobile app:
 
-### Manage the Viam Agent
+{{< tabs >}}
+{{% tab name="Mobile app" min-height="703px" %}}
```

Comment:
I'm unclear on the mobile app flow. Previously (when I last used it) you could only use it to provision NEW machines. If that's still the case, then the reference below to where configuration is for machines that already exist is pointless (but also unlikely to be harmful.) I'm fine with it either way, so I'll leave this section for @ale7714 to sign off on.

Reply:
I'll remove it.

Reply:
for now, will actually test this in the next week